README.md

```diff
@@ -6,74 +6,52 @@
 
-* [Node.js](http://nodejs.org/)
-* [npm](https://www.npmjs.org/)
-
-### npm dependencies
-
-These will be resolved automatically by simply running `npm install` on the directory of the project.
-
-* [express](https://github.com/strongloop/express)
-* [ejs](https://github.com/tj/ejs)
-* [Immutable.js](https://github.com/facebook/immutable-js/)
-* [Promise](https://github.com/then/promise)
-* [Moment](http://momentjs.com/)
-* [Specberus](https://github.com/w3c/specberus)
-* [`third-party-resources-checker`](https://github.com/dontcallmedom/third-party-resources-checker)
-
-## How to get it up and running
+Then run the following commands with your favorite terminal:
 
 ```bash
-$ git clone git@github.com:w3c/echidna.git
-$ cd echidna
-$ cp config.js.example config.js
-$ npm install
-$ npm start
+git clone https://github.com/w3c/echidna.git
+cd echidna
+cp config.js.example config.js
+npm install
 ```
 
-Then simply open [`http://localhost:3000`](http://localhost:3000) on your web browser and start throwing publication requests at it.
+## Usage
 
-Note: local setup of the full system is not supported currently due to dependency on W3C's DB and IPP system.
+*Note: local setup of the full system is not supported currently due to dependency on W3C's DB and IPP system, but having mock services that emulate these pieces is our short-term goal.*
 
-## Syntax and command-line parameters
+In your terminal, run the following:
 
-(with npm >= 2)
-```
-$ npm start [-- STAGING_PATH [HTTP_BASE_URI [PORT [RESULT_PATH]]]]
+```bash
+npm start [-- STAGING_PATH [HTTP_LOCATION [PORT [RESULT_PATH]]]]
 ```
 
-Meaning of positional parameters:
+You may use the optional defined below:
 
 1. `STAGING_PATH`: path in the local filesystem where documents will be downloaded; *staged*.
 (Default `/var/www/html/trstaging/`.)
-2. `HTTP_LOCATION`: HTTP endpoint for [Specberus](https://github.com/w3c/specberus) and the [Third-party checker](https://github.com/dontcallmedom/third-party-resources-checker).
+2. `HTTP_LOCATION`: HTTP endpoint for [Specberus](https://github.com/w3c/specberus) and the [Third Party Resources Checker](https://github.com/dontcallmedom/third-party-resources-checker).
 (Default `http://localhost/trstaging/`.)
 3. `PORT`: where Echidna will be listening for publication requests.
 (Default `3000`.)
 4. `RESULT_PATH`: local path where Echidna will dump the results of publication requests in JSON format.
 
-Examples:
+Note that they are only supported by npm 2 and above. Alternatively, you can use the configuration file `config.js`.
 
-```bash
-$ npm start -- /home/nick/public_html/staging/ http://localhost/~nick/staging/ 80
-```
-
-```bash
-$ npm start -- ../tmp/echidna-files/
-```
+Once the server is started, you can throw publication requests at it through a `curl`/`POST` request to its enpoint, [`http://localhost:3000/api/`](http://localhost:3000/api/), or using the web-based test based (described below).
 
 ## Automated testing
 
```

Comment:
It would be helpful if there was just a short description of what is being tested and why. I know it might be self evident to those building echidna, but I was like "will this test my spec? do I need to run this in relation to my spec?"